src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
4K 10-bit with a bitrate between 140 and 400 MB/s, well, that a lot whatever is your i7...
Of course, i could have lazzily downscaled it while transcoding.

And yes, many other way to lower the CPU use, i'm still waiting the user feedback.
Though i'm not skilled as you are.

The samples are taken from [here](http://jell.yfish.us/).
Thank you for the offer, any help is always welcome :wink: 
I'm sure you already have your ideas, but if you miss some, i've posted a few [there](http://www.universalmediaserver.com/forum/viewtopic.php?f=5&t=10283&p=32411#p32411).

But the more important point is if PS4 does even support fragmented MP4 files ? i'm waiting user answer while he will have tested my files. Because if not, MP4 will not be sustainable as a transcoding format  for PlayStation. And i read on a forum that one guy reported that it doesn't support it.